### PR TITLE
feat: add route(chart) for openrice.comOpenrice

### PR DESCRIPTION
**Pull request by after9**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/openrice/zh/hongkong/explore/chart/most-bookmarked
/openrice/zh/hongkong/explore/chart/best-rating
/openrice/zh/hongkong/explore/chart/most-popular
/openrice/zh/hongkong/explore/chart/best-dessert
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/openrice/namespace.ts [hunk 1]
```diff
@@ -5,5 +5,5 @@ export const namespace: Namespace = {
     url: 'www.openrice.com',
     categories: ['shopping'],
     description: '美食網站Openrice相关資訊',
-    lang: 'zh-HK',
+    lang: 'zh',
```
Comment (TonyRL) [inline lib/routes/openrice/namespace.ts:8]:
```suggestion
    lang: 'zh-HK',
```
`zh` is not listed in https://github.com/DIYgod/RSSHub/blob/fd5a888f185d97c7679dfb724a63b31b6de51638/lib/types.ts#L93

File: lib/routes/openrice/chart.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    handler,
+    categories: ['shopping'],
+    example: '/openrice/zh/hongkong/explore/chart/most-bookmarked',
+    parameters: { lang: '语言，缺省为 zh', category: '类别，缺省为 most-bookmarked' },
+    name: '香港餐廳排行榜',
+    description: `
+  | 简体 | 繁體 | EN |
+  | ----- | ------ | ----- |
+  | zh-cn | zh | en |
+  <br/>
```
Comment (TonyRL) [inline lib/routes/openrice/chart.ts:22]:
```suggestion

```

File: lib/routes/openrice/chart.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    maintainers: ['after9'],
+    handler,
+    categories: ['shopping'],
+    example: '/openrice/zh/hongkong/explore/chart/most-bookmarked',
+    parameters: { lang: '语言，缺省为 zh', category: '类别，缺省为 most-bookmarked' },
+    name: '香港餐廳排行榜',
+    description: `
+  | 简体 | 繁體 | EN |
+  | ----- | ------ | ----- |
+  | zh-cn | zh | en |
```
Comment (TonyRL) [inline lib/routes/openrice/chart.ts:21]:
```suggestion
  | zh-cn | zh | en |

```

**Review by after9 — COMMENTED** [review]
updated, please check again. Thanks!

File: lib/routes/openrice/chart.ts [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  | most-bookmarked | best-rating | most-popular | best-dessert |
+  `,
+};
+
+async function handler(ctx) {
+    const lang = ctx.req.param('lang') ?? 'zh';
+    const category = ctx.req.param('category') ?? 'most-bookmarked';
+
+    const urlPath: string = `/${lang}/hongkong/explore/chart/${category}`;
+    const response = await ofetch(baseUrl + urlPath, {});
```
Comment (TonyRL) [inline lib/routes/openrice/chart.ts:34]:
```suggestion
    const response = await ofetch(baseUrl + urlPath);
```

**Pull request merged**